apps/api/src/services/index.ts

```diff
@@ -87,8 +90,21 @@ class ServiceRegistry {
   private readonly _eventsRepository: EventsRepository;
   private readonly _eventProcessor: EventProcessor;
   private readonly _boostAwardService: BoostAwardService;
+  private readonly _privyClient: PrivyClient;
 
   constructor() {
+    // Initialize Privy client (use MockPrivyClient in test mode to avoid real API calls)
+    if (config.server.nodeEnv === "test") {
+      this._privyClient = new MockPrivyClient(
+        config.privy.appId,
+        config.privy.appSecret,
+      ) as unknown as PrivyClient;
+    } else {
+      this._privyClient = new PrivyClient(
+        config.privy.appId,
+        config.privy.appSecret,
+      );
+    }
```

Comment:
This is pretty strange. I'm no vitest expert, but isn't the idea that, in your test file, you mock the import for `import { PrivyClient } from "@privy-io/server-auth";` and then this service registry is using the mocked object?